app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  has_many :groups, dependent: :destroy
+  has_many :entities, dependent: :destroy
+
+  # validates :name, presence:true
```

Comment:
- [ ] Please remove commented-out code as it is of no use, makes your code look cluttered, and takes up space